TLM/TLM/UI/MainMenu/ClearTrafficButton.cs

```diff
@@ -14,13 +14,11 @@ public class ClearTrafficButton : BaseMenuButton {
 
         public override void SetupButtonSkin(AtlasBuilder futureAtlas) {
             // Button backround (from BackgroundPrefix) is provided by MainMenuPanel.Start
-            this.Skin = new U.ButtonSkin() {
-                Prefix = "ClearTraffic",
-                BackgroundPrefix = "RoundButton",
-                BackgroundHovered = true,
-                BackgroundActive = true,
-                ForegroundActive = true,
-            };
+            this.Skin = ButtonSkin.CreateSimple(
```

Comment:
indentation

Reply:
Looks correct to me? 
![bild](https://user-images.githubusercontent.com/288724/128426477-5bc113ef-10a0-4cef-831b-f5afabd54c2c.png)